scripts/yg-privacyenhanced.py

```diff
@@ -15,7 +15,7 @@
 """THESE SETTINGS CAN SIMPLY BE EDITED BY HAND IN THIS FILE:
 """
 
-ordertype = 'swreloffer'  # [string, 'swreloffer' or 'swabsoffer'] / which fee type to actually use
+ordertype = 'swreloffer'  # [string, 'swreloffer', 'swabsoffer', 'sw0reloffer', 'sw0absoffer'] / which fee type to actually use
```

Comment:
Default to `sw0reloffer`? I think you forgot, since it looks right in the yg-basic below.

Reply:
Btw this brings up another point, we should just dynamically check that whatever the user chose fits the wallet type (do an `isinstance` check somewhere in the yg startup).

Reply:
I had the same thought few weeks ago, just forgot to comment here. Yes, why do we need reloffer/swreloffer/sw0reloffer at all and not just check wallet type?

Reply:
> I had the same thought few weeks ago, just forgot to comment here. Yes, why do we need reloffer/swreloffer/sw0reloffer at all and not just check wallet type?

+1 , I see no reason not go drop the setting and just set it from the wallet type.

Reply:
The reason why I left it to `swreloffer` is that I assume that in the first few months of the switch, it would be better that people who want extra privacy use nested segwit for better anon set? How does that sound?

Reply:
`ordertype` incompatible with wallet type will not work anyway, we can use reloffer/swreloffer/sw0reloffer in JM protocol, just there is no need to have different settings in yield generator conf.

Reply:
@jules23 ah ok, i get it, it wasn't a mistake. but as I and @kristapsk have discussed here, we should just remove this because of the wallet dependency;  I can add it in another commit after the PR, if that's less bother.

I suppose, to note, we still need to leave the user the 'abs'/'rel' distinction, so it's not just removing the option, it's changing it, and then adding the dynamic choice ('abs' -> 'sw0absoffer') in code.

Reply:
Addressed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/656/commits/733cf38e76b86ebe432b9c8e7677aa0f85a0295c - I believe this commit does exactly what we agreed here, let me know if you don't agree.